flexmeasures/api/v3_0/assets.py

```diff
@@ -184,6 +186,110 @@ def index(
 
         return response, 200
 
+    @route(
+        "/<id>/sensors",
+        methods=["GET"],
+    )
+    @use_kwargs(
+        {
+            "asset": AssetIdField(data_key="id"),
+        },
+        location="path",
+    )
+    @use_kwargs(
+        {
+            "page": fields.Int(
+                required=False, validate=validate.Range(min=1), default=1
```

Comment:
I saw this in the terminal debug when running tests:

```
home/nicolas/envs/fm-3.10/lib/python3.10/site-packages/marshmallow/fields.py:1182: RemovedInMarshmallow4Warning: The 'default' argument to fields is deprecated. Use 'dump_default' instead.
```

I guess we should adapt this line and line 205. And then test manually if the page still works as expected.

Reply:
page loads fine

Reply:
do I implement this across other API's? (maybe in a different PR)

Reply:
Did you test not sending `per_page` to see if it falls back on 10 as expected?

Reply:
this was an already existing API so it will return a non-pagianted list if `page` is not passed

Reply:
oh, I get it now, but to test that in the test_case we will need more data, there are only 3 data for assets right now

Reply:
No I meant to test this manually.

Reply:
This actually works manually I've tested it, below is a screenshot of me hitting the API without passing the `per_page` query. if you look to the right the amount of data in the `data` key is 10 an the total record is 14

![Screenshot from 2024-10-04 10-41-01](https://github.com/user-attachments/assets/c2c90a4e-811e-4541-ab93-9634a2a096c8)